packages/ibm-products/src/components/EditInPlace/EditInPlace.tsx

```diff
@@ -27,7 +33,101 @@ const defaults = {
   size: 'sm',
 };
 
-export let EditInPlace = forwardRef(
+type Size = 'sm' | 'md' | 'lg';
+type AlignPropType =
+  | 'top'
+  | 'top-left'
+  | 'top-right'
+  | 'bottom'
+  | 'bottom-left'
+  | 'bottom-right'
+  | 'left'
+  | 'right';
+type Shape = {
+  cancel: AlignPropType;
+  edit: AlignPropType;
+  save: AlignPropType;
+};
+
+interface EditInplaceProps extends PropsWithChildren {
+  /**
+   * label for cancel button
+   */
+  cancelLabel: string;
+  /**
+   * By default the edit icon is shown on hover only.
+   */
+  editAlwaysVisible?: boolean;
+  /**
+   /**
```

Comment:
You have an extra `/**` on this line and the indentation is tad bit off. 🤔 I would have expected the precommit to have caught the indentation but running `yarn format` should fix it.

Reply:
pre-commit hook is running for me but not sure why its not catching this.  Also I noticed that when I do yarn format, its not formatting the tsx file. So I am doing format document.

Resolved 👍

Reply:
Ahh good catch and I see @szinta opened a PR to fix this as well 🥳